src/export/StructureDefinitionExporter.ts

```diff
@@ -1258,6 +1316,10 @@ export class StructureDefinitionExporter implements Fishable {
    */
   export(): Package {
     this.checkInvariants();
+    this.typeCharacteristicCodes = this.fishForFHIR(
+      'http://hl7.org/fhir/type-characteristics-code',
+      Type.CodeSystem
+    ) as CodeSystem;
```

Comment:
I think `export()` only gets called once, and then it loops through all the SDs on line 1324. So it's probably OK, but I still think it's worth considering initializing it in the constructor since it's a private member of the class.